README.md

```diff
@@ -1,2 +1,104 @@
-# resource_library
-Developed by Coding Ninjas
+# Resource Library
+> Connecting students to instructors.   
+
```

Comment:
When I think about Resource Library, I think the overall objective is providing resources (events, articles, links, etc.) to help students learn. So perhaps we can change line 2 to "A central location for resources for students to learn more and develop as a software engineer through articles, links, and events". :) Something like this, but please feel free to update my wording.